src/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
This should be `netcoreapp2.1`. Note that `netcoreapp` and `netstandard` are different things.  We want `netcoreapp2.1` for any and all executables and tests. We want `netstandard2.0` for any libraries.

Reply:
Moreover, we don't even want this in the lib folder as it is just a commandline tool.